modules/apps/data-engine/data-engine-service/src/main/java/com/liferay/data/engine/internal/service/DEDataRecordCollectionServiceImpl.java

```diff
@@ -480,6 +483,24 @@ public DEDataRecordCollectionSaveResponse execute(
 		}
 	}
 
+	@Override
```

Comment:
@mclomello Remove deDataRecordCollectionSearchExecutor instance.
```suggestion
	@Override
	public DEDataRecordCollectionSearchResponse execute(
			DEDataRecordCollectionSearchRequest deDataRecordCollectionSearchRequest)
		throws DEDataRecordCollectionException {

		try {
			return deDataRecordCollectionSearchExecutor.execute(
				deDataRecordCollectionSearchRequest);
		}
		catch (Exception e) {
			throw new DEDataRecordCollectionException(e);
		}
	}
```

Reply:
Changed and fixed according last comment.